examples/tutorials/tutorial_2d_crm.py

```diff
@@ -1,13 +1,10 @@
 import flow360 as fl
-from flow360.component.simulation.operating_condition.operating_condition import (
-    operating_condition_from_mach_reynolds,
-)
 from flow360.component.simulation.unit_system import SI_unit_system, u
-from flow360.examples import Tutorial_2dcrm
+from flow360.examples import Tutorial2DCRM
 
 fl.Env.preprod.active()
 
-project = fl.Project.from_file(Tutorial_2dcrm.geometry, name="Tutorial 2D CRM from Python")
+project = fl.Project.from_file(Tutorial2DCRM.geometry, name="Tutorial 2D CRM from Python")
```

Comment:
Is this example the same as in the https://github.com/flexcompute/Flow360/pull/572? If so, please check the comments in that PR.

Reply:
I will do that